### ISSUE TITLE
Publish AndrOBD to Google Play store

### ISSUE THREAD

**Issue by aha999** [fr3ts0n/AndrOBD#245]
**Is your feature request related to a problem? Please describe.**
- Currently, AndrOBD is only available on F-Droid. 
- AndrOBD needs contributors to the source code, design etc. so it can grow faster

**Describe the solution you'd like**

- I think it would be best if the project was uploaded to [Google Play store](https://play.google.com/store/apps), but as a paid app. Many open source products go in this direction, so the program is free on F-Droid, but paid on Google Play, so it can support the developer.
- I think AndrOBD has everything it needs to be published, but i would wait until a new GUI is developed.
- proposed price =1,99€

**Comment by highroller** [comment]
I agree. It should be published in play store as a free app. A lot of open source apps are published in play store, why not this one. 

I don't ever side load apps, it prevents me from installing this one.
